builtin/settingtypes.txt

```diff
@@ -138,7 +138,7 @@ invert_mouse (Invert mouse) bool false
 #    Mouse sensitivity multiplier.
 #
 #    Requires: keyboard_mouse
-mouse_sensitivity (Mouse sensitivity) float 0.2 0.001 10.0
+mouse_sensitivity (Mouse sensitivity) float 0.2 0.000000001 1000000.0
```

Comment:
honestly 0.001 seems very reasonable to me
we could make the upper limit the inverse, so 1000

Reply:
The default is not 1, so I don't think it needs to be symmetric.

The thing is, the limit is arbitrary. Imo the limits should roughly represent what the engine can easily handle, and not add unnecessary strictness.

With 0.001, and my mouse's highest dpi setting (roughly 10'000 dpi) I can do a full horizontal rotation in only 10 seconds.
With the new limit, I seemingly can't move it at all, like it's supposed to be.